wp-content/themes/ngisweden/functions/shortcodes/deployed_tools_versions.php

```diff
@@ -3,10 +3,13 @@
 // NGI Deployed Tools Versions
 // Renders the file generated on the cluster during deployment,
 // showing which versions of which tools are currently in production.
+[ngisweden_deployed_tools_versions_sthlm deployed_file_name='deployed_tools.sthlm.version']
+[ngisweden_deployed_tools_versions_upps deployed_file_name='deployed_tools.upps.version']
```

Comment:
No need to have two different shortcodes here, can just have one that takes the varying argument.

Also, I guess that you meant these to be comment lines?

```suggestion
// [ngisweden_deployed_tools_versions deployed_file_name='deployed_tools.sthlm.version']
// [ngisweden_deployed_tools_versions deployed_file_name='deployed_tools.upps.version']
```

Reply:
I'm not sure how the arguments are passed to the shortcode otherwise? Would these be read if they are commented out?

Reply:
If these lines are kept I get:

`Parse error: syntax error, unexpected 'deployed_file_name' (T_STRING), expecting ']' in /var/www/html/wp-content/themes/ngisweden/functions/shortcodes/deployed_tools_versions.php on line 6`

Reply:
Yeah, it's invalid PHP as it is - just an array sat there doing nothing.

@aanil the shortcode is written like this within the WordPress page content, not within the source code. Arguments are passed in the `$atts_raw` associative array which the function gets as an argument.

Reply:
Ah right, I thought it was in the file itself. I wasn't sure about passing arguments, so I added the locations as an array and looped through it to read the different files.

Reply:
Not sure that looping is a great idea, as then we get just the one table and there might be different versions of similar tools..? 🤔

Reply:
There would be two different tables as I append to output in the loop, I think. I'll test it out now.

Reply:
ah ok, sorry - whitespace is a bit tough to read in the diff..